From 45366d2500116c0b745a5b97f20a15c624eedf11 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Tue, 13 May 2008 18:53:39 +0100
Subject: [PATCH] config-add-back-MMC_UNSAFE_RESUME.patch

After LL debug disable, "unsafe" resume allows resume from SD card
rootfs without corruption, WAH

Signed-off-by: Andy Green <andy@openmoko.com>
---
 defconfig-2.6.24 |    2 +-
 defconfig-gta02  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/defconfig-2.6.24 b/defconfig-2.6.24
index 4df10bf..d091489 100644
--- a/defconfig-2.6.24
+++ b/defconfig-2.6.24
@@ -1483,7 +1483,7 @@ CONFIG_SDIO_S3C24XX_DMA=y
 CONFIG_SDIO_AR6000_WLAN=y
 CONFIG_MMC=y
 # CONFIG_MMC_DEBUG is not set
-# CONFIG_MMC_UNSAFE_RESUME is not set
+CONFIG_MMC_UNSAFE_RESUME=y
 
 #
 # MMC/SD Card Drivers
diff --git a/defconfig-gta02 b/defconfig-gta02
index 4d040ff..1fa2d80 100644
--- a/defconfig-gta02
+++ b/defconfig-gta02
@@ -1395,7 +1395,7 @@ CONFIG_SDIO_S3C24XX=y
 CONFIG_SDIO_S3C24XX_DMA=y
 CONFIG_SDIO_AR6000_WLAN=y
 CONFIG_MMC=y
-CONFIG_MMC_DEBUG=y
+# CONFIG_MMC_DEBUG is not set
 CONFIG_MMC_UNSAFE_RESUME=y
 
 #
-- 
1.5.6.5

